### PR TITLE
test: don't count hard-to-test lines for coverage

### PR DESCRIPTION
**Pull request by nfischer**
No change in logic.

Add `/* istanbul ignore next */` lines for hard-to-test lines so that
they don't count against us during code coverage.

I've also adjusted comments that I found confusing, and changed some
formatting.

Partial fix for #671

### CODE REVIEW

File: src/common.js [hunk 1]
```diff
@@ -367,11 +369,13 @@ function wrap(cmd, fn, options) {
           if (e.msg === 'earlyExit') {
             retValue = e.retValue;
           } else {
+            /* istanbul ignore next */
```
Comment (freitagbr) [inline src/common.js]: I think `/* istanbul ignore else */` above the `if` statement also works.

Comment (nfischer) [inline src/common.js]: Yeah, either way. I just usually stick with `ignore next`. Do you think I should change it?

Comment (freitagbr) [inline src/common.js]: With `ignore next`, I think istanbul will mark the `else` route as not taken. `ignore else` should improve test coverage slightly.

Comment (nfischer) [inline src/common.js]: I'll try it tonight and see if I see a difference in either line or branch coverage

Comment (nfischer) [inline src/common.js]: @freitagbr done. Same line coverage, but `ignore else` slightly improves branch coverage.

**Comment by nfischer** [comment]
@freitagbr PTAL. I also moved one `ignore next` above the `if` so we get slightly better branch coverage. Branch coverage isn't a priority right now though, so let's just focus on line coverage.

**Comment by freitagbr** [comment]
LGTM

**Pull request merged**